woocommerce/woocommerce-ios
# [Product Multi-Selection] Remove experimental feature toggle

Closes: #9335 

## Description
This PR removes the existing toggle button to switch on/off Product Multi-Selection for developer builds within the Experimental Features screen. By removing the toggle, product multi-selection will be used by default on dev environments.

The feature remains hidden from merchants behind the `.productMultiSelectionM1` feature flag.

## Testing instructions
- Run the app
- Go to Menu > Settings > Experimental Features. See "Product Multi-Selection" is not an option.
- Go to Orders >  Tap `+` > `+ Add Products` > See that products can be tapped on a multi-selection basis

| Before | After  |
|---|---|
| <img width=450 src="https://user-images.githubusercontent.com/3812076/224233779-4a1522d4-4610-42f8-98c6-5cecf2aabee3.png"> | <img width=450 src="https://user-images.githubusercontent.com/3812076/229552420-80a21135-97ad-41a5-8fab-7489f0ae0ec0.png"> |
